src/components/index.ts

```diff
@@ -11,6 +11,7 @@ export * from "./Form";
 export * from "./Icon";
 export * from "./IconButton";
 export * from "./Image";
+export * from "./Layout";
```

Comment:
alphabetized! nice 👍 

BUT! This is not tested yet, so you shouldn't export it yet. Recommend commenting it out so that we know it's _almost_ ready to be deployed. 
```suggestion
// export * from "./Layout";
```

Reply:
commented out